src/lib/draw/CollapsibleCard.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+  // TODO Lots of overlap with AccordionItem
```

Comment:
Mostly just copied. We could refactor, or pick a better component library (possibly govuk based)